subsys/bluetooth/Kconfig

```diff
@@ -16,6 +16,12 @@ config BT_MAX_CONN
 	range 1 20 if BT_LL_SOFTDEVICE
 	default 2 if BT_LL_SOFTDEVICE && BT_CENTRAL && BT_PERIPHERAL
 
+config BT_LL_SOFTDEVICE_HEADERS_INCLUDE
+	bool
+	default y if !BT_LL_SOFTDEVICE && !BT_CTLR
```

Comment:
You can remove !BT_LL_SOFTDEVICE as BT_LL_SOFTDEVICE depends on BT_CTLR

Reply:
done